src/table/pagination-table/components/head/HeadCell.tsx

```diff
@@ -33,9 +34,10 @@ const HeadCell = ({ column, columnIndex, columnsLength }: HeadCellProps) => {
   const columnWidths = useContextSelector(TableContext, (value) => value.columnWidths);
   const isSelectionMode = useContextSelector(TableContext, (value) => value.baseProps.selectionsAPI?.isModal());
   const setFocusedCellCoord = useContextSelector(TableContext, (value) => value.setFocusedCellCoord);
+  const focusedCellCoord = useContextSelector(TableContext, (value) => value.focusedCellCoord);
```

Comment:
This we don't want to do in this way. `focusedCellCoord` updates eveytime you move the focus -> every head cell will rerender. What you could do is to check if the current cell is focused, so you actually get a value that is a bool:
```suggestion
const isThisCellFocused = useContextSelector(
    TableContext,
    (value) => value.focusedCellCoord[0] === 0 && value.focusedCellCoord[1] === columnIndex,
  );
```
Of course you can break out this logic if you want